tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -82,20 +83,28 @@ def _navigate(self, result: Mapping[str, Any]) -> Any:
         return result
 
     @command
-    def print(
-        self,
-        context: str,
-        background: Optional[bool] = None,
-        margin: Optional[Mapping[str, Any]] = None,
-        orientation: Optional[str] = None,
-        page: Optional[Mapping[str, Any]] = None,
-        page_ranges: Optional[List[str]] = None,
-        scale: Optional[float] = None,
-        shrink_to_fit: Optional[bool] = None
-    ) -> Mapping[str, Any]:
-        params: MutableMapping[str, Any] = {
-            "context": context
-        }
+    def reload(self,
+               context: str,
+               ignoreCache: Optional[bool] = None,
```

Comment:
Hey @thiagowfx, CI fails because `ignoreCache` should be snake-cased, so rather `ignore_cache`

Reply:
Ah, thanks! Fixed

I assume the BiDi param still needs to be [`ignoreCache`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-reload) though, correct?

Just updated the py arguments to `ignore_cache`.